src/test/java/unstable/ContextMenuTests.java

```diff
@@ -98,4 +97,57 @@ public void test3() {
         sleep(EVENT_DELAY);
     }
 
+    @Test
+    public void addToWatchListPanel_createNewWatchListPanel_issueAddedToNewPanel() {
+        rightClick("#dummy/dummy_col0_9");
+        sleep(EVENT_DELAY);
+        click("Add to watch list");
+        sleep(EVENT_DELAY);
+        click("New watch list");
+        type("Watch List #1");
+        press(KeyCode.ENTER);
+
+        ListPanel watchList1 = find("#dummy/dummy_col1");
+        assertEquals("Watch List #1", watchList1.getCurrentInfo().getPanelName());
+        assertEquals("id:dummy/dummy#9", watchList1.getCurrentInfo().getPanelFilter());
+    }
+
+    @Test
+    public void addToWatchListPanel_addToExistingWatchListPanel_issueAddedToExistingPanel() {
+        click("Panels");
+        click("Create");
+        ListPanel createdPanel = find("#dummy/dummy_col1");
+        createdPanel.setPanelName("New Panel");
+        rightClick("#dummy/dummy_col0_11");
+        click("Add to watch list");
+        click("New Panel");
+        assertEquals("id:dummy/dummy#11", createdPanel.getCurrentInfo().getPanelFilter());
+    }
+
+    @Test
+    public void addToWatchListPanel_issueFromDifferentRepos_addedIssuesRetainRepoInfo() {
+        rightClick("#dummy/dummy_col0_9");
+        sleep(EVENT_DELAY);
```

Comment:
I see. I was just following the other tests in the unit test file, will fix that.